src/edu/gatech/cs2340/trydent/ContinuousEvent.java

```diff
@@ -0,0 +1,59 @@
+package edu.gatech.cs2340.trydent;
+
+/**
+ * Represents an event that is called by the engine every frame.
+ * @author Garrett Malmquist
+ *
+ */
+public abstract class ContinuousEvent {
```

Comment:
Is this how the animations are going to be implemented internally?
Also, if the animation objects aren't going to be subclassing this, we may want to leverage the singleton or a dispatcher so that the users won't have to wrap everything into a ContinousEvent subclass.